articles/fleet-4.17.0.md

```diff
@@ -0,0 +1,88 @@
+
+[Read more about bringing Orbit out of beta.](https://github.com/fleetdm/fleet/issues/5454)
+
+## Engage with your users through Fleet Desktop.
+**Available in Fleet Free and Fleet Premium**
+
+![Engage with your users through Fleet Desktop.](<../website/assets/images/articles/fleet-4.17.0-2-1600x900@2x.jpg>)
+Fleet desktop is also out of beta! Show end users exactly what Fleet can see on their device, whether they have vulnerable software, and their policy scorecard. All they need to do is click on the Fleet logo in their system tray. If you're using a Zero Trust framework, Fleet Desktop will help users avoid losing access to internal tools by giving them the ability to proactively resolve vulnerabilities and stay compliant with your organization's security policies. 
+
+Today, Fleet Desktop is a great way to empower end-users to be active participants in securing their devices. Our vision for the future is that Fleet Desktop will offer open communications channels and build trust between users and IT and Security practioners. We're also continuing to focus on adding features that will help end-users take ownership of their device's health and provide tools to help them resolve issues quickly and easily.
+
+To include Fleet Desktop in your installer package, add the `--fleet-desktop` flag to your `fleetctl package` command. 
+
+[Read more about bringing Fleet Desktop out of beta.](https://github.com/fleetdm/fleet/issues/5684)
+
+## Battery condition added to Host Details for MacOS.
+**Available in Fleet Free & Fleet Premium**
+
+![Battery condition added to Host Details.](../website/assets/images/articles/fleet-4.17.0-3-1600x900@2x.jpg)
+Quickly determine whether a MacOS host's battery needs to be replaced. The physical health of your hosts is every bit as important as security and Fleet makes it easy to see how well the battery is performing. 
+
+
+## More new features, improvements, and bug fixes
+
+In 4.17.0, we also:
+
+* Added native support for M1 Macs.
+
+* Improved reporting of error states on the health dashboard and added separate health checks for MySQL and Redis with `/healthz?check=mysql` and `/healthz?check=redis`.
+
+* Improved SSO login failure messaging.
+
+* Fixed osquery tables that report incorrect platforms.
+
+* Added `docker_container_envs` table to the osquery table schema on the **Query* page.
+
+* Updated Fleet host detail query so that the `os_version` for Ubuntu hosts reflects the accurate patch number.
+
+* Improved accuracy of `software_host_counts` by removing hosts from the count if software has been uninstalled.
+
+* Improved accuracy of the `last_restarted` date. 
+
+* Fixed `/api/_version_/fleet/hosts/identifier/{identifier}` to return the correct value for `host.status`.
```

Comment:
```suggestion
* Fixed `/api/v1/fleet/hosts/identifier/{identifier}` to return the correct value for `host.status`.
```

Suggestion to use `v1` here. If this makes sense, can you please also update the CHANGELOG entry for this?

Reasoning on waiting to include (publicly announce) the new API versioning is here: https://github.com/fleetdm/fleet/pull/6137#issuecomment-1175530387